src/create_gmad.cpp

```diff
@@ -104,8 +105,20 @@ namespace CreateAddon
 
 		Output::Msg( "Writing file list...\n" );
 
+		uint64_t iFileListSize = 0;
+		BOOTIL_FOREACH( f, files, String::List )
+		{
+			iFileListSize = iFileListSize + 4 + 180 + 8 + 4; // File number (4) + File name (180 file name length limit) + File size (8) + File CRC (4)
```

Comment:
I am not sure I like this seemingly arbitrary 180 character amount. The file list writing is not that performance intensive anyway.

Reply:
I added it so that the buffer would hopefully be enough for the whole file list, or else it would just reallocate the buffer again every time something is written.